### PR TITLE
adds ternaries to set start and end index for pagination

### PR DESCRIPTION
**Pull request by edleeks87**
# Description
Fixes issue where page sizes were all the same so records exceeded the total records value

### CODE REVIEW

File: src/components/table/table.stories.js [hunk 1]
```diff
@@ -26,12 +26,12 @@ const handleChange = (e, tableOptions) => {
...
 };
 
-const buildRows = (pageSizeFromKnobs) => {
+const buildRows = (pageSizeFromKnobs, totalRecords) => {
   const pageSize = pageSizeFromKnobs;
   const currentPage = store.get('currentPage');
 
-  const endIndex = pageSize * currentPage;
-  const startIndex = endIndex - pageSize;
+  const endIndex = pageSize * currentPage <= totalRecords ? pageSize * currentPage : totalRecords;
```
Comment (dquilter) [inline src/components/table/table.stories.js]: Could create a variable for `pageSize * currentPage`

File: src/components/table/table.stories.js [hunk 2]
```diff
@@ -26,12 +26,12 @@ const handleChange = (e, tableOptions) => {
...
+  const startIndex = totalRecords === endIndex ? totalRecords - (endIndex % pageSize) : endIndex - pageSize;
```
Comment (andrewjtait) [inline src/components/table/table.stories.js]: Maybe this is a bit clearer?
```
let endIndex = pageSize * currentPage;
if (endIndex > totalRecords) endIndex = totalRecords;

const currentPageSize = (endIndex === totalRecords) ? (endIndex % pageSize) : pageSize;
const startIndex = endIndex - currentPageSize
```

File: src/components/table/table.stories.js (hunk 1 above)
Comment (edleeks87) [reply src/components/table/table.stories.js]: Yeah I've gone and added it

File: src/components/table/table.stories.js (hunk 2 above)
Comment (edleeks87) [reply src/components/table/table.stories.js]: Gone for a halfway between your way and mine

**Pull request merged**